build/deploy-build.ts

```diff
@@ -4,7 +4,7 @@ import { packages } from './config';
 
 const deploy = createBuilder([
   ['Deploy builds', tasks.publishToRepo],
-  ['Deploy docs', tasks.publishDocs],
+  // ['Deploy docs', tasks.publishDocs],
```

Comment:
@brandonroberts do you think this is still required?

Reply:
Maybe. We use it for `https://next.ngrx.io` but I don't know that its heavily used

Reply:
Ah I see, I haven't thought of that.
Would you be OK with doing that in a next PR? After we're sure that the current nightlies are working.

Reply:
Sure